src/styles/card.css

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  text-align: center;
+  width: 200px;
+  border: 1px solid transparent;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
```

Comment:
checklist  point
<img width="593" alt="image" src="https://user-images.githubusercontent.com/102411130/228154834-3c22c12d-1304-46b4-9ccd-2baa27ce3e78.png">

if you want to center img horizontally, use `margin: 0 auto` or value you need instead of `0` (vertical margins)